docker-compose.yml

```diff
@@ -1,7 +1,18 @@
 version: "3.9"
 
 services:
+  backend:
+    build:
+      context: .
+      dockerfile: apps/backend/Dockerfile
+    depends_on:
+      - mongodb
+    ports:
+      - 3000:3000
+    environment:
+      - MONGODB_URI=mongodb://mongo:mongo@animavita_mongo/?authMechanism=DEFAULT
```

Comment:
NIT: you could use env files for that

https://towardsdatascience.com/a-complete-guide-to-using-environment-variables-and-files-with-docker-and-compose-4549c21dc6af?gi=3540cd2edfc1